Tools/LyTestTools/ly_test_tools/mars/filebeat_client.py

```diff
@@ -51,7 +53,7 @@ def _open_socket(self):
         try:
             self._socket.connect((self._filebeat_host, self._filebeat_port))
         except (ConnectionError, socket.timeout):
-            raise FilebeatExn("Failed to connect to Filebeat") from None
+            raise exceptions.LyTestToolsFrameworkException(FilebeatExn("Failed to connect to Filebeat")) from None
```

Comment:
This appears to intentionally be a separate exception. I see no reason to wrap it.